### ISSUE TITLE
Reproducing results in Table H.1 of "Does Zero-Shot Reinforcement Learning Exist?" 

### ISSUE THREAD

**Issue by enjeeneer** [facebookresearch/controllable_agent#5]
Hi guys. Thanks very much for open-sourcing your implementation, and for the great work!

I have a quick question about how you produced the results reported in Table H.1 of your most recent paper _"Does Zero Shot Reinforcement Learning Exist?"_. I'm aware you train your model for $10^6$ gradient steps; are the results in Table H.1 produced by evaluating after the final gradient step, or do you evaluate at a checkpoint prior to the final step? If it's the latter, how do you decide on the checkpoint?

Thanks very much in advance!

**Comment by ahmed-touati** [comment]
Hi @enjeeneer,
Thanks for your interest.
The evaluation reported in the table is done after the the final gradient step.

**Comment by enjeeneer** [comment]
Hi @ahmed-touati thanks for your quick reply. 

To clarify why I'm asking: when I reproduce your results on `walker` using the `proto` buffer, I find that models from checkpoints prior to the final gradient step perform similarly, and sometimes better, than the results in Table H.1. Whereas the model from the final gradient step performs slightly worse, though I haven't been able to average across 10 seeds which likely explains any differences. 

Did you ever evaluate performance on earlier gradient steps?

Thanks again!

**Comment by ahmed-touati** [comment]
Hi @enjeeneer,
This is possible, we used only  RND dataset to finetune hyper-parameters and we used the same of hyperparams for APS and Proto buffers.
We observed that sometimes the performance decreases a bit over training. 
Decreasing the learning rate and training for longer could be helpful.
We didn't want to perform early-stopping for each env and buffer.  We prefered to run multiple seeds and average the overall performance at the final step.

**Comment by bofusun** [comment]
Hello, if you reproduce the fb with goal_space setting? I meet the problem of poor performance with command:

python -m url_benchmark.train_online agent=fb_ddpg task=walker_walker load_replay_buffer=./walker/rnd/replay.pt 

If the performance of table H.1 is produced with goal_space=simplied_walker or I reproduced it with wrong way?
Could you give me any help?
Thanks!
